.github/workflows/run_periodic_tests.yml

```
@@ -0,0 +1,128 @@
+# Run all unit tests and integration tests for all Python versions 
+# and platforms at 3am UTC every day and on PRs to the main branch
+name: PyBaMM
```

Maybe something like -
```suggestion
name: Scheduled tests
```

---

I agree this would be more informative, but I think it might also change the name in the badge we have in the README. Not sure if there is a way to have another name in the badge.

---

Maybe we can rename it to just `CI`

---

We can maybe name it just "Scheduled"?

---

Yes, that works